AndreTkachenkoOrg/lfg-bot
# Add multi channel support

# Description

Allow users to setup and use several lfg channels at the same time.

Resolves #25 

## Type of change


# How Has This Been Tested?

1. Create 2 lfg channels
2. Create event via `lfg setup` command in 1st channel
3. Create event via 'lfg setup' command in 2nd channel
4. Ensure both channels handled calls correctly and created events.

**Test Configuration**:

Firmware version: 0.0.307
Hardware: Win 10
Toolchain: VSCode, Discord, Google Chrome

# Checklist:
